<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Viewer</class>
 <widget class="QMainWindow" name="Viewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/src/Chess/icon.ico</normaloff>:/src/Chess/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="menuBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1252</width>
      <height>700</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/backcolor.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/start_btn-2.png</normaloff>:/src/start_btn-2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>500</width>
      <height>220</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/title.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sideManifest">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>100</y>
      <width>150</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/Redside.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="processTime">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>30</y>
      <width>200</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe Arabic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
    <property name="digitCount">
     <number>8</number>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Filled</enum>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="loadRecord">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/loading_btn.png</normaloff>:/src/loading_btn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>520</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/exit_btn.png</normaloff>:/src/exit_btn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLCDNumber" name="blackTime">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>260</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="redTime">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>380</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="limitMode">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/limit.png</normaloff>:/src/limit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="normalMode">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/normal.png</normaloff>:/src/normal.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="blackFrame">
    <property name="geometry">
     <rect>
      <x>775</x>
      <y>255</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/blackTimeFrame.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="redFrame">
    <property name="geometry">
     <rect>
      <x>775</x>
      <y>375</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/redTimeFrame.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="winer">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>380</width>
      <height>100</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/redWin.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="oneMore">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/again.png</normaloff>:/src/again.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="returnMenu">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/returnMeun.png</normaloff>:/src/returnMeun.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="winBackground">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>420</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/overBackground.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="blackGiveUp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/giveUp.png</normaloff>:/src/giveUp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="redGiveUp">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>556</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/src/giveUp.png</normaloff>:/src/giveUp.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="check">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>275</y>
      <width>130</width>
      <height>130</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/redCheck.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="danceGif2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>170</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/yooooo.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="danceGif">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>520</y>
      <width>170</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/src/yooooo.gif</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>menuBackground</zorder>
   <zorder>startButton</zorder>
   <zorder>title</zorder>
   <zorder>sideManifest</zorder>
   <zorder>processTime</zorder>
   <zorder>loadRecord</zorder>
   <zorder>exit</zorder>
   <zorder>blackTime</zorder>
   <zorder>redTime</zorder>
   <zorder>limitMode</zorder>
   <zorder>normalMode</zorder>
   <zorder>blackFrame</zorder>
   <zorder>redFrame</zorder>
   <zorder>winBackground</zorder>
   <zorder>winer</zorder>
   <zorder>returnMenu</zorder>
   <zorder>oneMore</zorder>
   <zorder>blackGiveUp</zorder>
   <zorder>redGiveUp</zorder>
   <zorder>check</zorder>
   <zorder>danceGif2</zorder>
   <zorder>danceGif</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
